nuget/nunit.runners.xamarin.nuspec

```diff
@@ -43,6 +43,9 @@ Supported Xamarin platforms:
         <file src="src\runner\nunit.runner.wp81\bin\Release\nunit.runner.wp81.dll" target="lib\wpa81\nunit.runner.wp81.dll" />
         <file src="src\runner\nunit.runner.wp81\bin\Release\nunit.runner.wp81.xml" target="lib\wpa81\nunit.runner.wp81.xml" />
         
+        <file src="nuget\uap10.0\MainPage.xaml.txt.pp" target="content\uap10.0\MainPage.xaml.txt.pp" />
+        <file src="nuget\uap10.0\MainPage.xaml.cs.txt.pp" target="content\uap10.0\MainPage.xaml.cs.txt.pp" />
```

Comment:
I'm look at it now, hold on this PR. I also saw this

Severity	Code	Description	Project	File	Line
Warning		The SDK "SQLite.UAP.2015, Version=3.9.2" depends on the following SDK(s) "Microsoft.VCLibs, version=14.0", which have not been added to the project or were not found. Please ensure that you add these dependencies to your project or you may experience runtime issues. You can add dependencies to your project through the Reference Manager.	SQLite.Net.Tests.UWP